### PR TITLE
fix args handling in merge driver, app

### PR DESCRIPTION
**Pull request by minrk**

### CODE REVIEW

File: nbdime/nbmergeapp.py [hunk 1]
```diff
@@ -48,12 +48,12 @@ def main_merge(args):
     else:
         nbdime.log.debug("Merge completed successfully with no unresolvable conflicts.")
 
-    if args.decisions:
+    if decisions:
```
Comment (martinal) [inline nbdime/nbmergeapp.py]: No, args.decisions is deliberate, it shouldn't print decisions without --decisions on the commandline.

File: nbdime/nbmergeapp.py [hunk 2]
```diff
@@ -48,12 +48,12 @@ def main_merge(args):
...
         # Print merge decisions (including unconflicted)
         out = io.StringIO()
         pretty_print_merge_decisions(b, decisions, out=out)
         nbdime.log.warning("Conflicts:\n%s", out.getvalue())
-    elif mfn:
+    if mfn:
```
Comment (martinal) [inline nbdime/nbmergeapp.py]: This was also deliberate, if --decisions are asked for, it doesn't print the merged notebook.

File: nbdime/nbmergeapp.py (hunk 1 above)
Comment (minrk) [inline nbdime/nbmergeapp.py]: ah, ok. Then there's a problem because `args.decisions` is undefined.

**Comment by minrk** [comment]
Thanks for the clarification, @martinal! I figured out that the problem was merge args being added in the wrong place. I've moved them to the common merge_args location, so they are shared by the various merge apps.

**Comment by minrk** [comment]
Oops, wrong again, it was just in the mergedriver. I've made the smaller fix. I'm working on another PR for greatly increased CLI coverage, but this should fix the mergedriver for now.

**Comment by martinal** [comment]
Great, we need that.

**Pull request merged**